tests/client/core/test_utils.js

```diff
@@ -516,7 +512,7 @@ describe('loadCategoriesIfNeeded', () => {
     return loadCategoriesIfNeeded(props).then(() => {
       assert(dispatch.calledWith(action), 'dispatch not called');
       mockApi.verify();
-      mockActions.verify();
+      return mockActions.verify();
```

Comment:
Seems a bit weird to return the last one of multiple calls. Is this needed to pass lint? If so perhaps this one should be excluded instead.